image/image_test.go

```diff
@@ -249,3 +254,68 @@ func TestCheckFileInfo(t *testing.T) {
 		t.Error("sha512 should not be matched")
 	}
 }
+
+func TestUncompressedContentSize(t *testing.T) {
+	contentSize := int64(2 * 1024)
+
+	dir, err := createTestDir("testDir1", contentSize)
+	if err != nil {
+		t.Fatalf("Can't create test dir: %v", err)
+	}
+
```

Comment:
Remove empty line here.

Reply:
I can't because the linter shows a warning `only one cuddle assignment allowed before defer statement (wsl)`